references/plugins/def-doc/component/alibaba-ack.eg.md

```diff
@@ -0,0 +1,14 @@
+```yaml
```

Comment:
Here markdown documents need manual maintenance?

Reply:
yes, examples need to be provided manully